bot/cogs/source_code_cog.py

```diff
@@ -181,7 +181,7 @@ def chunk_iterable(self, iterable, chunk_size):
     
     def process_source(self, source: str, line_start: int = None, line_stop: int = None):
         split_source = [f'{i:03d} |  {value}' for i, value in enumerate(source.splitlines())]
-        filtered_source = split_source[line_start or 0: line_stop or len(source)]
+        filtered_source = split_source[line_start or 0: line_stop+1 if line_stop is not None else len(source)]
```

Comment:
this can just be

if not line_stop else

Reply:
gotcha, i'll change it. also working on making lines print from 1 instead of 0.

Reply:
when i change it to if not line_stop else it gives me the TypeError again